src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -54,7 +81,8 @@ public void robotPeriodic() {
 
     @Override
     public void autonomousInit() {
-        subsystems.drivebaseSubsystem.resetPose(RigidTransform2.ZERO);
+        //subsystems.drivebaseSubsystem.resetPose(new Pose2d(RigidTransform2.ZERO) );
+        subsystems.drivebaseSubsystem.resetPose(GeoConvertor.rigidToPose(RigidTransform2.ZERO));
         SimplePathBuilder builder = new SimplePathBuilder(new Vector2(0, 0), Rotation2.fromDegrees(0));
         subsystems.drivebaseSubsystem.follow(builder.lineTo(new Vector2(20, 0)).lineTo(new Vector2(20, 20))
                 .lineTo(new Vector2(0, 20)).lineTo(new Vector2(0, 0)).build());
```

Comment:
Should this be commented out? It uses the 2910 path builder.

(I think this was the one we talked about that was running the square path instead of the WPILib path follower, someone correct me if I'm wrong).